ts/components/menuAndSettingsHooks/useProBadgeOnClickCb.tsx

```
@@ -287,7 +260,5 @@ export function useProBadgeOnClickCb(
     return showNoCb;
   }
 
-  assertUnreachable(context, 'useProBadgeOnClickCb: context not handled');
```

can we keep this one there? Or it was causing an issue?
I'd much rather have a false compilation issue and add a new case that having it fail silently

---

Yeah ill add it back and resolve, i think its doable